examples/dt/uni01alpha/control-plane/service-values.yaml

```diff
@@ -0,0 +1,140 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinderVolumes:
+    lvm-iscsi:
+      replicas: 1
+      nodeSelector:
+        openstack.org/cinder-lvm: ""
+      customServiceConfig: |
+        [lvm]
+        image_volume_cache_enabled = false
+        volume_driver = cinder.volume.drivers.lvm.LVMVolumeDriver
+        volume_group = cinder-volumes
+        target_protocol = iscsi
+        target_helper = lioadm
+        volume_backend_name = lvm_iscsi
+        target_ip_address=172.18.0.5
+        target_secondary_ip_addresses = 172.19.0.5
+
+  cinderBackup:
+    customServiceConfig: |
+      [DEFAULT]
+      backup_driver = cinder.backup.drivers.swift.SwiftBackupDriver
+    replicas: 1
+
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = default_backend:swift
+
+      [glance_store]
+      default_backend = default_backend
+
+      [default_backend]
+      swift_store_create_container_on_put = True
+      swift_store_auth_version = 3
+      swift_store_auth_address = {{ .KeystoneInternalURL }}
+      swift_store_endpoint_type = internalURL
+      swift_store_user = service:glance
+      swift_store_key = {{ .ServicePassword }}
+    default:
+      replicas: 1
+
+  ironic:
+    enabled: true
+
+  swift:
+    enabled: true
+
+  octavia:
+    enabled: true
+
+  heat:
+    enabled: true
+
+  telemetry:
+    enabled: false
```

Comment:
Disabled for now...

Reply:
It would be better to just disable autoscaling so the metricStorage and the ceilometer features are already tested

Reply:
ceilometer deployment seems good but metric storage is throwing

```
2024-04-01T14:18:27Z	INFO	Controllers.MetricStorage	Can't own MonitoringStack resource	{"controller": "metricstorage", "controllerGroup": "telemetry.openstack.org", "controllerKind": "MetricStorage", "MetricStorage": {"name":"metric-storage","namespace":"openstack"}, "namespace": "openstack", "name": "metric-storage", "reconcileID": "3e745745-c639-484a-9652-01df514ffd12"}
```

Reply:
Yes, that is being thrown because cluster-observability-operator is not deployed in openshift, which is a pre-requisite. However, we just give info about it and wait for it to appear.

Can cluster-observability-operator be deployed on the cluster before running the DT?

Reply:
Yes, we should as it is a prerequisite. Let me add the necessary artifacts.

Reply:
For reference, we deploy COO in our CI like this: https://github.com/openstack-k8s-operators/telemetry-operator/blob/main/ci/create-coo-subscription.yaml, just for if it is useful here.